app/templates/authenticated/security/files.hbs

```diff
@@ -1,2 +1,2 @@
-{{page-title "All Files"}}
-<FileSearchList @project={{@model}}></FileSearchList>
+{{page-title 'All Files'}}
+<Security::FileSearchList @projectId={{@model.projectid}} />
```

Comment:
add new line between page title and component and also why `@model.projectid` ?

Reply:
The `projectid` is from the route.